### PR TITLE
corrected typos to 2 READMEs

### PR DESCRIPTION
**Pull request by markjdugger**
Corrected formatting typos in 2 READMEs

### CODE REVIEW

**Comment by markjdugger** [comment]
@tschie Please approve these formatting errors in 2 READMEs

**Review by tschie — APPROVED** [review]

**Comment by markjdugger** [comment]
@tschie Please review additional changes, which are in commit f06b3eb.

- Mostly minor formatting correction.
- Major: I changed all ordered sublists from `<ol>` to `<ul>` because ordered sublists do not display properly on Chrome, IE, Edge. At least, not after our nice HTML README gets transformed into HTML where all `<`become `&lt;` and all `>` become `&gt;`  I'm not sure why the transform (which translates XML from the CMS topic to HTML for the website) does this. The easy fix, however is to use `<ul>` sublists which display fine.
- Here is example of one I did *not* change yet:
https://master-dev.developers.arcgis.com/java/latest/sample-code/feature-layer-rendering-mode-scene-.htm. Under *How it works*, look at text following 2b.

_Below are links to the modified topics on master-dev, in case you want to visually verify any items:_
•	Feature Collection Layer, in How it works section
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/feature-collection-layer.htm> 

•	Feature Layer Rendering Mode (Map), in How it works section
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/feature-layer-rrendering-mode-map-.htm> 

•	List Related Features, too much spacing in snippet in How it works section
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/list-related-features.htm> 
 
•	Query Map Image Sublayer, snippet is blank. Also: *How to use sample:* has an extraneous colon.
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/query-map-image-sublayer.htm> 
 
•	Tile Cache, in How ti work, wrong use of code format in step 4
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/tile-cache.htm> 
 
•	Unique Value Renderer, extra vertical space in snippet
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/unique-value-renderer.htm> 
 
•	Convex Hull List, under Relevant API, types should not be in code format. Just plain text.
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/convex-hull-list.htm> 
 
•	Blend Renderer, How it works, step 5, bullet point needs to be indented
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/blend-renderer.htm> 
 
•	Local Server Dynamic Workspace Shapefile, step 6, final bullet point needs to be indented
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/dynamic-workspace-shapefile.htm> 
 
•	Animate 3d Graphic, under How to use the sample, correct colons and hypens.  Out of control
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/animate-3d-graphic.htm> 

•	Change Atmosphere Effect, under Relevant API, types should NOT use code format
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/change-atmosphere-effect.htm> 
 
•	Distance Measurement Analysis, under Tags, terms should be comma-separate list
From <https://master-dev.developers.arcgis.com/java/latest/sample-code/distance-measurement-analysis.htm>

File: src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_shapefile/README.md [hunk 1]
```diff
@@ -17,15 +17,15 @@
 <ol>
 <li>Create and run a local server.
 <ul><li><code>LocalServer.INSTANCE</code> creates a local server</li></ul>
-<ul><li><code>Server.startAsync()</code> starts the server asynchronously</li></ul>
+<ul><li><code>Server.startAsync()</code> starts the server asynchronously</li></ul></li>
```
Comment (tschie) [inline src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_shapefile/README.md]: I think the `ul` tags are wrong. Should be
```html
<li>Create and run a local server.
<ul>
<li><code>LocalServer.INSTANCE</code> creates a local server</li>
<li><code>Server.startAsync()</code> starts the server asynchronously</li>
</ul>
```

**Review by tschie — APPROVED** [review]
Looks good. One one thing looks like it needs fixing. Thanks for doing this!

File: src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_shapefile/README.md (hunk 1 above)
Comment (markjdugger) [reply src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_shapefile/README.md]: Fixed per your recommendation.

**Pull request merged**